terraform/modules/rds/main.tf

```diff
@@ -0,0 +1,56 @@
+# Non-compliant RDS configuration (missing encryption, backups, and monitoring)
+resource "aws_db_instance" "main" {
+  identifier        = "${var.environment}-db"
+  engine           = "postgres"
+  engine_version   = "13.7"
+  instance_class   = var.instance_class
+  allocated_storage = 20
+
+  # Non-compliant: Credentials in plain text
+  name     = var.db_name
+  username = "admin"  # Hardcoded credentials
+  password = "insecure123!"  # Hardcoded password
+
+  # Non-compliant: Public access enabled
+  publicly_accessible = true
+
+  # Non-compliant: No encryption
+  storage_encrypted = false
```

Comment:
The RDS instance should have a backup retention period of at least 30 days to comply with SOC2 requirements. Update `backup_retention_period` to a minimum of 30.